polaris-react/src/tokens/token-groups/color.light.json

```diff
@@ -110,6 +110,8 @@
   "surface-success-subdued-hovered": "rgba(236, 246, 241, 1)",
   "surface-success-subdued-pressed": "rgba(226, 241, 234, 1)",
   "text-success": "rgba(0, 128, 96, 1)",
+  "icon-attention": "rgba(138, 97, 22)",
+  "surface-attention": "rgba(255, 234, 138)",
```

Comment:
I hesitate to add it to `legacy-tokens` because we should be slowly removing values from that file, not adding to it. This is a color that is important to keep in the system (see https://github.com/Shopify/polaris/pull/4840) and not accidentally remove again so that's why it seemed like the color token group is a good place for it? Is that not the primary place we are keeping color tokens now?

Reply:
> we should be slowly removing values from that file, not adding to it

Ideally, but it sounds like these values shouldn't have been deleted and would have otherwise been placed in the `legacy-tokens` file during the v9 release. 

> Is that not the primary place we are keeping color tokens now?

Yes, however adding these tokens to the `color.*` files indicates these are public facing tokens that folks are welcome to start including in their styles. Was there any input from design to validate the semantic naming of `icon-attention` and `surface-attention`? I feel like we should start being very strict and considerate when adding new tokens moving forward as updates require breaking changes.

It's very likely I'm missing some context, so feel to disregard my concerns and ship these tokens 👍